code/modules/mob/living/blood.dm

```diff
@@ -32,10 +32,10 @@
 // Takes care blood loss and regeneration
 /mob/living/carbon/human/handle_blood(delta_time, times_fired)
 
-	if(NOBLOOD in dna.species.species_traits || bleedsuppress || (HAS_TRAIT(src, TRAIT_FAKEDEATH)))
+	if(NOBLOOD in dna.species.species_traits || bleedsuppress || (HAS_TRAIT(src, TRAIT_FAKEDEATH)) && !iszombie(src))
```

Comment:
Any reason this can't be a defined trait? While that works, the existing method with traits wraps it up neatly.

Reply:
Running it this way completely removes zombies from blood_handling, which includes but is not limited to...
Bleeding, integrating blood, wounds, blood-drain from other effects, etc.

It was the previous way iszombie was ran and caused issues down the line.

Reply:
This still doesn't explain why this is a species check and not something else